bokehjs/src/coffee/core/properties.coffee

```diff
@@ -107,9 +110,6 @@ export class Property
 
     @init()
 
-    if trigger
-      @trigger("change")
```

Comment:
@bryevdv, I tried hard to figure out what's the purpose of this. Looks like a leftover, maybe from the time when `Property` was a backbone model. Anyway, no tests failed after removing this.